Sources/HipMobileUI.Droid/HipMobileUI.Droid.csproj

```diff
@@ -36,9 +36,9 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <AndroidUseSharedRuntime>True</AndroidUseSharedRuntime>
+    <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
```

Comment:
These two values have been changed (maybe unintentionally?) They can be reverted to `True` for the first one and `False` for the second.

Reply:
yes,seems like my local problem. will switch it right after.